src/style.css

```diff
@@ -1,3 +1,64 @@
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav__list {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 50px;
```

Comment:
instead of add margins for last and first (logo) items in the header better to add horizontal padding for header